nyc_taxi/nyc_taxi-nongeo.ipynb

```diff
@@ -0,0 +1,225 @@
+    "ranged.opts(width=800, height=500)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we're using the default histogram-equalized color mapping function to reveal density differences across this space.  If we used a linear mapping (same code as above wirh `normalization='linear'`), we can mainly see that there are a lot of values near the origin, but all the rest are colored the same minimum (defaulting to light blue) color:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scatter_ds = hd.spread(hd.datashade(hv.Scatter(df, 'trip_distance', 'fare_amount'), normalization='linear'))\n",
+    "labelled = scatter_ds.redim.label(trip_distance=\"Distance, miles\", fare_amount=\"Fare, $\")\n",
+    "ranged = labelled.redim.range(trip_distance=(0, 20), fare_amount=(0,40))\n",
+    "ranged.opts(width=800, height=500)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Fares are discretized to the nearest 50 cents, making patterns less visible, but there is both an upward trend in tips as fares increase (as expected), but also a large number of tips higher than the fare itself, which is surprising:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scatter_ds = hd.spread(hd.datashade(hv.Scatter(df, 'fare_amount', 'tip_amount')))\n",
+    "labelled = scatter_ds.redim.label(tip_amount=\"Tip, $\", fare_amount=\"Fare, $\")\n",
+    "labelled.redim.range(tip_amount=(0,40), fare_amount=(0,20))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Interestingly, tips go down when the number of passengers is greater than 1:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scatter_ds = hd.spread(hd.datashade(hv.Scatter(df, 'passenger_count', 'tip_amount'), x_sampling=0.15))\n",
+    "labelled = scatter_ds.redim.label(passenger_count=\"Passengers\", fare_amount=\"Fare, $\")\n",
+    "labelled.redim.range(tip_amount=(0,60), passenger_count=(-0.5,6.5))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we've reduced the resolution along the x axis so that instead of getting isolated points for this inherently discrete data, you can see more-visible horizontal line segments.\n",
+    "\n",
+    "The above plots all use Bokeh directly, but a much wider range of interactive plots can be built easily using the separate [HoloViews](http://holoviews.org) library, which builds Bokeh and Matplotlib plots from high-level specifications.  For instance, Datashader currently only provides 2D aggregates, but you can easily make a zoomable one-dimensional histogram using HoloViews to dynamically collapse across a second dimension:"
```

Comment:
This comment is out of date, but presumably you're relying on there being a whole pass done once pyviz-topics is complete?